presto-main/src/main/java/com/facebook/presto/util/Reflection.java

```diff
@@ -27,6 +27,14 @@
 {
     private Reflection() {}
 
+    /**
+     * Returns a MethodHandle to get the specified field.
+     * <p>
+     * Warning: The way Oracle JVM implements producing MethodHandle to get a field involves
```

Comment:
Are you positive calling `Class#getField` involves `MethodHandle`-s?

(I though you mean cost involved if i later `unreflect` returned `Field`.. but I couldn't find the usages.)